src/dbnode/storage/types.go

```diff
@@ -581,6 +594,19 @@ type databaseShard interface {
 
 	// DocRef returns the doc if already present in a shard series.
 	DocRef(id ident.ID) (doc.Document, bool, error)
+
+	// AggregateTiles does large tile aggregation from source shards into this shard.
+	AggregateTiles(
+		ctx context.Context,
+		reader fs.DataFileSetReader,
+		sourceNsID ident.ID,
+		sourceBlockStart time.Time,
+		sourceShard databaseShard,
+		opts AggregateTilesOptions,
+		wOpts series.WriteOptions,
+	) (int64, error)
+
+	latestVolume(blockStart time.Time) (int, error)
```

Comment:
`latestVolume` -> `LatestVolume` for consistency.

Reply:
Will also address in one of downstream PRs.